modules/administration/pages/monitoring.adoc

```diff
@@ -182,7 +176,7 @@ See https://github.com/wrouesnel/postgres_exporter.
 
 Installing and configuring exporters is done using a Salt formula.
 
-
+When you have the exporters installed and configured, you can manually configure monitoring, or set up service discovery so that Prometheus automatically starts scraping metrics from systems as they are enabled.
```

Comment:
Here it does not really get clear how someone would `manually configure monitoring` exactly, I guess. Maybe it could be replaced with `configure Prometheus to scrape the exposed metrics`, even though this gives only a little bit more information?

Reply:
The idea is that there are several procedures below that statement: for manually configuring, and for service discovery. I'll update the wording to make that clearer.